tensorflow/src/TensorFlow/Nodes.hs

```diff
@@ -101,18 +100,12 @@ instance a ~ () => Fetchable ControlNode a where
 instance Nodes (Tensor v a) where
     getNodes t = Set.singleton <$> getOrAddOp (t ^. tensorOutput . outputOp)
 
-fetchTensorList :: TensorType a => Tensor v a -> Build (Fetch (Shape, [a]))
-fetchTensorList t = fmap (fmap V.toList) <$> fetchTensorVector t
-
 fetchTensorVector :: forall a v . TensorType a
-                  => Tensor v a -> Build (Fetch (Shape, V.Vector a))
```

Comment:
This isn't an instance. I was thinking of `fetchTensorVector` as an internal function and you can still get the shape by using `TensorData`. Adding instance for `(Shape, ...)` sounds like a good idea though.